src/test/java/seedu/address/logic/commands/EditPersonDescriptorTest.java

```diff
@@ -64,7 +64,8 @@ public void toStringMethod() {
                 + editPersonDescriptor.getName().orElse(null) + ", phone="
                 + editPersonDescriptor.getPhone().orElse(null) + ", email="
                 + editPersonDescriptor.getEmail().orElse(null) + ", address="
-                + editPersonDescriptor.getAddress().orElse(null) + ", tags="
+                + editPersonDescriptor.getAddress().orElse(null) + ", birthday="
+                + editPersonDescriptor.getBirthday().orElse(null) + ", tags="
                 + editPersonDescriptor.getTags().orElse(null) + "}";
```

Comment:
Should we standardize on whether to put birthday or tags in front or it doesnt really matter?

Reply:
Previously I encountered some bug, and I thought tags have to be at the back because tags can have multiple values so I put birthday before tags, but turns out it doesn't matter. So let's standardize and have new attributes be at the back.